pkg/Library/NodeGraph/NodeInspector.js

```diff
@@ -161,9 +161,9 @@ async constructConnectedStore(connection, selected, pipeline, nodeTypes, service
   const value = await Promise.all(selected?.map(
     async ({from, store}) => {
       const node = pipeline.nodes.find(node => node.key == from);
-      const nodeType = this.findNodeType(node?.name, nodeTypes);
-      if (nodeType) {
-        return await this.getBindingValue(store, nodeType.$stores[store], node, service);
+      if (node) {
```

Comment:
could you, please, revert this change? this is a PR i made earlier this week and it didn't merge well, i think

Reply:
better?